CollisionAlert.lua

```diff
@@ -14,9 +16,9 @@ function Instance:onInit()
 		self.collisionModel = getEditor():createNew(self:getParent():getObjectKit(), "CollisionModel")	
 	end
 
-	local icd = getEditor():createNew(self.collisionModel:getCollisionDetectors(), "ImpactCollisionDetector")
+	icd = getEditor():createNew(self.collisionModel:getCollisionDetectors(), "ImpactCollisionDetector")
```

Comment:
Would prefer self.icd to match the other styling (in each place it's used)

Reply:
I wondered about this.  I thought what if there were multiple objects in the scene using the mod.  Each should be instanced throughout yea?

Reply:
Yeah it will be either way, I think